### ISSUE TITLE
Fix navigation links in Windows build of site

### ISSUE THREAD

**Issue by claremacrae** [claremacrae/claremacrae.github.io#9]
* site.github.is_project_page is True
* site.github.is_user_page

This means that the links on the left hand side are for a project page, not the customised links to Blog and About, that I was expecting to see

**Comment by claremacrae** [comment]
This may help: https://github.com/jekyll/github-metadata/blob/master/docs/configuration.md

**Comment by claremacrae** [comment]
This too: https://help.github.com/en/articles/repository-metadata-on-github-pages

**Comment by claremacrae** [comment]
Running the following command....

`bundle exec jekyll build --verbose`

... includes the following output:

```
   GitHub Metadata: Generating for claremacrae/claremacrae.github.io
   GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.
```

Which may explain the problem.

**Comment by claremacrae** [comment]
https://github.com/DIYbiosphere/sphere/issues/140
-> 
https://github.com/github/pages-gem/issues/399

**Comment by claremacrae** [comment]
Adding this to my `_config.yml` did not make the warning go away.

`github: [metadata]`

**Comment by claremacrae** [comment]
This is old, but may help: http://blog.johannesmp.com/2017/02/13/fixing-jekyll-serve-on-windows/

**Comment by claremacrae** [comment]
Some progress:
Following the **Fix “The GitHub API credentials you provided aren’t valid.”** section of this page got rid of that warning when building the site:

http://blog.johannesmp.com/2017/02/13/fixing-jekyll-serve-on-windows/

* So now `site.github.is_project_page` appears to be false
* But unfortunately `site.github.is_user_page` is also false

So I now get no links at all in the navigation area.

**Comment by claremacrae** [comment]
This is now fixed - see d8612ae2ab57649b5b2bb48904668146da603c66
